packages/jetpack-logo/src/Assets_Manager_Interface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Jetpack\Assets;
+
+/**
+ * Interface Assets_Manager_Interface
+ */
+interface Assets_Manager_Interface {
+	/**
+	 * Assets_Manager_Interface constructor.
+	 *
+	 * @param array $assets_directory
+	 */
+	public function __construct( $assets_directory );
```

Comment:
This doesn't seem very clean - what will the `$assets_directory` contain eventually? Can we make it more specific and verbose?